src/overlay/Peer.cpp

```diff
@@ -1007,7 +1067,14 @@ Peer::maybeSendNextBatch()
             auto& timer = front.mMessage->type() == SCP_MESSAGE
                               ? om.mOutboundQueueDelaySCP
                               : om.mOutboundQueueDelayTxs;
-            timer.Update(mApp.getClock().now() - front.mTimeEmplaced);
+            auto& maxDelay = front.mMessage->type() == SCP_MESSAGE
+                                 ? mPeerMetrics.mMaxOutboundQueueDelaySCP
+                                 : mPeerMetrics.mMaxOutboundQueueDelayTxs;
+            auto const& diff = mApp.getClock().now() - front.mTimeEmplaced;
+            timer.Update(diff);
+            maxDelay = std::max(
```

Comment:
Same thing with the max here. Ideally, we can display some recent value to get a sense of how the peer is doing.

Reply:
Updated to show P99s from the last 30-second window

Reply:
Edit: 30 -> 300 seconds based on the offline discussion.